app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java

```diff
@@ -77,18 +95,35 @@ private void setUpNavigationDrawer() {
             changeFragment(nextFragment);
 
             item.setChecked(true);
-            toolbar.setTitle(item.getTitle());
             drawerLayout.closeDrawers();
+            setUpNewFragment(item.getTitle(), item.getItemId());
+
             return true;
         });
     }
 
+    /**
+     * Sets up the new fragment.
+     * @param title Title to display in the top bar
+     * @param id Id of the navigation item
+     */
+    private void setUpNewFragment(CharSequence title, int id) {
+        toolbar.setTitle(title);
+
+        if (id == R.id.navigationMyPets) {
+            floatingActionButton.show();
+        }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213265)